PhysicsTools/PatAlgos/python/producersLayer1/jetUpdater_cff.py

```diff
@@ -3,13 +3,13 @@
 from PhysicsTools.PatAlgos.recoLayer0.jetCorrections_cff import *
 from PhysicsTools.PatAlgos.producersLayer1.jetUpdater_cfi import *
 
-patJetCorrFactorsUpdated = patJetCorrFactors.clone(
+updatedPatJetCorrFactors = patJetCorrFactors.clone(
```

Comment:
why this name is changed? which doesn't seem to be necessary since the contents of this module is not changed.. This change of module name will require all physics analyses that need to re-applying the new 76x JEC using this module to do some unnecessary change of their analysis codes. 
For example, for our H->ZZ->4l analysis framework, the following line need to be changed if we want to change from 763patch2 to 764:
https://github.com/CJLST/ZZAnalysis/blob/miniAOD_76X/AnalysisStep/test/MasterPy/ZZ4lAnalysis.py#L1007

but this doesn't seem to be necessary...